### PR TITLE
Enabled GPTQ test and doc fix

### PR DESCRIPTION
**Pull request by quic-amitraj**
1. Unbaled GPTQ test
2. Doc fixed

### CODE REVIEW

**Review by quic-akuruvil — APPROVED** [review]

File: docs/source/validate.md [hunk 1]
```diff
@@ -29,7 +29,9 @@
 | [Jais-adapted-70b](https://huggingface.co/inceptionai/jais-adapted-70b) | ✔️ |
 | [Jais-adapted-13b-chat](https://huggingface.co/inceptionai/jais-adapted-13b-chat) | ✔️ |
 | [Jais-adapted-7b](https://huggingface.co/inceptionai/jais-adapted-7b) | ✔️ |
-
+| [granite-20b-code-base](https://huggingface.co/ibm-granite/granite-20b-code-base-8k) | ✔️ | ✔️ |
+| [granite-20b-code-instruct-8k](https://huggingface.co/ibm-granite/granite-20b-code-instruct-8k) | ✔️ | ✔️ |
+| [Starcoder1-15B](https://huggingface.co/bigcode/starcoder) | ✔️ | ✔️ |
```
Comment (ochougul) [inline docs/source/validate.md:34]: Is this tested?

File: tests/transformers/models/test_causal_lm_models.py [hunk 2]
```diff
@@ -32,8 +32,7 @@
     "hakurei/gpt-j-random-tinier",
     "mistralai/Mixtral-8x7B-Instruct-v0.1",
     "TheBloke/TinyLlama-1.1B-Chat-v0.3-AWQ",  # AWQ model
-    #    "TheBloke/Llama-2-7B-Chat-GPTQ",  # GPTQ model -> Enable once GPTQ+ROPE
-    #    issue is resolved
+    "TheBloke/Llama-2-7B-GPTQ",  # GPTQ model
```
Comment (ochougul) [inline tests/transformers/models/test_causal_lm_models.py:35]: It doesn't make sense for this to be here, it's out of context of this PR.
Please make separate PR for such cases in future

**Review by ochougul — CHANGES_REQUESTED** [review]

File: docs/source/validate.md (hunk 1 above)
Comment (quic-amitraj) [reply docs/source/validate.md:34]: @quic-akuruvil

File: tests/transformers/models/test_causal_lm_models.py (hunk 2 above)
Comment (quic-amitraj) [reply tests/transformers/models/test_causal_lm_models.py:35]: Sure

File: docs/source/validate.md (hunk 1 above)
Comment (quic-akuruvil) [reply docs/source/validate.md:34]: Yes tested and verified

**Review by ochougul — APPROVED** [review]

**Pull request merged**